readthedocs/templates/sphinx/conf.py.conf

```diff
@@ -10,7 +14,7 @@ source_parsers = {
         }
 master_doc = 'index'
 project = u'{{ project.name }}'
-copyright = u'2016'
+copyright = datetime.now().year
```

Comment:
Is this expected to be a string somewhere?

Reply:
Looks like yes!

Reply:
Interesting that all the tests pass but this simple change breaks all the builds p: